tests/mail/tests.py

```diff
@@ -1478,3 +1478,23 @@ def test_server_stopped(self):
         backend.open()
         self.server.stop()
         backend.close()
+
+
+class SMTPBackendStoppedServerFailSilentlyTest(SMTPBackendTestsBase):
```

Comment:
Add a test class `SMTPNoServerTests(SimpleTestCase):` before `test_send_messages_after_open_failed` and put this test there. Then you don't need `self.server.stop()`.